__tests__/app.test.js

```diff
@@ -32,9 +33,23 @@ describe("/api/topics", () => {
         })
     })
 
-    test("GET 404: Should respond with a 404 if endpoint is incorrect", () => {
+    test("GET 404: Should respond with a 404 (Not found) if endpoint is incorrect", () => {
```

Comment:
this is a general 404 and not specific to the topics endpoint - this would do the same for any non-existent endpoint e.g. /api/bananas so doesnt need to be associated with the topics